src/react-apps/applications/runtime/src/components/GenericComponent.tsx

```diff
@@ -12,7 +12,7 @@ export interface IProvidedProps {
   id: string;
   type: string;
   textResourceBindings: any;
-  dataBinding: string;
+  dataBindings: string;
```

Comment:
any reason you used dataBindings and not dataModelBindings as defined in the formLayout? Should we not use the same naming conventions?

Reply:
Databinding was the name that was defined in the testData, but I agree that it's better to use dataModelBindings since that's the name from formLayout.
The converter doesn't converts dataModelBindings to databinding? @jezpoz